analyticsclient/tests/test_course_summaries.py

```diff
@@ -0,0 +1,69 @@
+import httpretty
+
+
+from analyticsclient.tests import ClientTestCase
+
+
+class CourseSummariesTests(ClientTestCase):
+
+    def setUp(self):
+        super(CourseSummariesTests, self).setUp()
+        self.base_uri = self.get_api_url('course_summaries/')
+        self.course_summaries_client = self.client.course_summaries()
+        httpretty.enable()
+
+    @httpretty.activate
+    def test_all_summaries_url(self):
+        """Course summaries can be called without parameters."""
+        httpretty.register_uri(httpretty.GET, self.base_uri, body='{}')
+        self.course_summaries_client.course_summaries()
+
+    @httpretty.activate
+    def test_courses_ids(self):
+        """Course summaries can be called with course IDs"""
+
+        def register_uri(course_ids):
+            httpretty.reset()
+            uri_template = '{uri}?course_ids={ids}'
+            uri = uri_template.format(uri=self.base_uri, ids=course_ids)
+            httpretty.register_uri(httpretty.GET, uri, body='{}')
+
+        course_ids = ['edx/demo/course', 'another/demo/course']
+
+        register_uri([course_ids[0]])
+        self.course_summaries_client.course_summaries(course_ids=[course_ids[0]])
+
+        register_uri(course_ids)
+        self.course_summaries_client.course_summaries(course_ids=course_ids)
```

Comment:
I like it!  I'll give the query arg assertion a shot.  The tests were relying on the client throwing an error if the generated URL didn't match the registered one.